x-pack/plugins/maps/server/maps_telemetry/maps_telemetry.test.js

```diff
@@ -85,16 +80,4 @@ describe('buildMapsSavedObjectsTelemetry', () => {
       mapsTotalCount: 5,
     });
   });
-
-  test('returns expected telemetry data from index patterns', async () => {
```

Comment:
not sure if we should be removing this test if we're fixing this. Perhaps add api-integration test on `stats` endpoint

Reply:
> not sure if we should be removing this test if we're fixing this

Yeah, things are more dependent on the index patterns service and a little less easy to mock in the current setup, but agreed it would be valuable to have this sort of test in place. I'll give it another look and see what I can come up with

> Perhaps add api-integration test on stats endpoint

Agreed this is needed but I think should be a separate issue since really our telemetry needs "bigger picture" testing for more than just the fields updated by this PR. I've created this issue to track: https://github.com/elastic/kibana/issues/91223

Reply:
Updated to include tests with index patterns service mocks!